### PR TITLE
Fix selected clippy lints

### PR DESCRIPTION
**Pull request by aljazerzen**

### CODE REVIEW

File: edb/edgeql-rust/src/normalize.rs [hunk 1]
```diff
@@ -40,21 +39,67 @@ pub enum Error {
...
+        value: var.into(),
+        start,
+        end,
+    });
+    res.push(CowToken {
+        kind: Kind::CloseParen,
+        value: ")".into(),
+        start,
+        end,
+    });
```
Comment (tailhook) [inline edb/edgeql-rust/src/normalize.rs:97]: This is definitely less readable.

File: edb/edgeql-parser/src/helpers.rs [hunk 2]
```diff
@@ -179,29 +183,59 @@ fn unquote_unicode_string() {
...
 }
 
 #[test]
 fn newline_escaping_str() {
-    assert_eq!(_unquote_string(r"hello \
-                                world").unwrap(), "hello world");
+    assert_eq!(
+        _unquote_string(
+            r"hello \
+                                world"
```
Comment (tailhook) [inline edb/edgeql-parser/src/helpers.rs:206]: This indentation doesn't matter. So doesn't have to be that weird.

**Comment by aljazerzen** [comment]
There are code chunks that have been improved, there are ones have been made less readable, but most importantly, the code style is uniform now.

I don't like a lot of what rustfmt does, but I like writing abominations of code and convert it easily into something acceptable. So this PR is here just make all this changes at once instead of formatting only files that I touch in upcoming commits.

**Comment by aljazerzen** [comment]
There are code chunks that have been improved, there are ones have been made less readable, but most importantly, the code style is uniform now.

I don't like a lot of what rustfmt does, but I like writing abominations of code and convert it easily into something acceptable. So this PR is here just make all this changes at once instead of formatting only files that I touch in upcoming commits.

**Comment by aljazerzen** [comment]
For any future use, this collection of lints is currently ignored in this repo:
```
cargo clippy -- -A clippy::collapsible_if -A clippy::extra_unused_lifetimes -A clippy::derive_partial_eq_without_eq -A clippy::zero_ptr -A clippy::manual_strip -A clippy::new_ret_no_self -A clippy::needless_return -A clippy::type_complexity -A clippy::needless_return -A clippy::manual_strip -A clippy::vec_init_then_push -A clippy::derive_partial_eq_without_eq -A clippy::while_let_on_iterator -A clippy::len_zero -A clippy::new_ret_no_self -A clippy::single_char_pattern -A clippy::zero_ptr -A clippy::manual_strip -A clippy::needless_return -A clippy::useless_format -A clippy::too_many_arguments -A clippy::clone_on_copy
```

**Review by tailhook — APPROVED** [review]

**Pull request merged**